core/ktor/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id("composenews.android.library")
+    libs.plugins.apply {
+        alias(kotlinx.serialization)
+    }
+}
+
+android {
+    namespace = "ir.composenews.ktor"
```

Comment:
This is a module about networking in general. I think `ktor` is very specific for this purpose. For example, this library may change with another networking library. So, I propose renaming `core/ktor` to a more proper name.

Reply:
I appreciate your insight. You're absolutely right, and I should have considered a more generic name from the start. I wasn't entirely sure about the naming when I began, but your point about potential future changes in networking libraries is spot-on. A more general name would indeed be more appropriate.
What do you think about renaming it to something like 'core-networking'? This would keep it generic while still indicating its purpose. Do you have any other suggestions for the module name that you think might fit better?